src/features/calendar/components/index.tsx

```diff
@@ -1,4 +1,4 @@
-import { Box } from '@mui/material';
+import { Box } from '@mui/system';
```

Comment:
This is the wrong `Box`. It should be from `@mui/material` (IIRC). It's super annoying and I've made the same mistake multiple times. I've learned to look out for `@mui/system` by now but can't remember the exact name of the correct module.